doc/sphinx/proof-engine/ltac.rst

```diff
@@ -766,27 +770,20 @@ Selects and applies the first tactic that solves each goal (i.e. leaves no subgo
 in a series of alternative tactics:
```

Comment:
The introduction of this paragraph seems slightly contradictory (and redundant) with the tactic's description. The correct explanation is the one in the tactic's description.

Reply:
Dropped it.  I think adding "(ie leaves no subgoal)" to elaborate on "solve the goal" doesn't really add much.